authors/apps/articles/exceptions.py

```diff
@@ -0,0 +1,21 @@
+from rest_framework.exceptions import APIException
+
+
+class NotFound(APIException):
```

Comment:
@myrdstom in the `restframework.exceptions` module there is already a `NotFound` exception why do you implement your own rather than use the one provided by `drf`

Reply:
Noted working on it, I should have resolved it earlier